nipy/algorithms/graph/tests/test_forest.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import print_function
+
```

Comment:
On Fri, 04 Sep 2015, Matthew Brett wrote:

> In [1]nipy/algorithms/graph/tests/test_forest.py:

> > @@ -1,3 +1,5 @@
> > +from __future__ import print_function
> > +

> Yes, but that works in any Python version, and is printing a tuple:

> $ python2.6 -c 'print("some", "strings")'
> ('some', 'strings')
> $ python2.7 -c 'print("some", "strings")'
> ('some', 'strings')
> $ python3.4 -c 'print("some", "strings")'
> some strings

> But maybe you meant - could I just print the tuple so it looks like it's a
> print function call?

that was just awesome!  Thank you matthew for bearing with me ;)